contracts/Management.sol

```diff
@@ -70,6 +74,24 @@ contract Management is IManagement, Ownable {
         emit NewFeeNumerator(_feeNumerator);
     }
 
+    /**
+        @notice Set referrer fee ratio
+        @dev Caller must be ADMIN and the referrer fee must not be greater than the overall fee
+        @param _feeNumerator the fee numerator
+     */
+    function setReferrerFeeRatio(uint256 _feeNumerator)
+        external
+        override
+        onlyOwner
+    {
+        if (_feeNumerator > FEE_DENOMINATOR || _feeNumerator > feeNumerator)
```

Comment:
Also it should validate that `ReferrerFee + OveralFee` must be less than FEE_DENOMINATOR since total fee can't be 100%. You should update this test case.